app/uk/gov/hmrc/fhregistrationfrontend/views/layout.scala.html

```diff
@@ -43,9 +44,17 @@
         isBrowserBackMimic: Boolean = false
 )(contentBlock: Html)(implicit request: RequestHeader, messages: Messages)
 
+@pageScripts = @{
```

Comment:
Only the `fh.js` file should be included by default in the pages.
The others should be added into the individual pages as needed.
Consider if autocomplete was added to a text input page that didn't need it - there could be potential regressions.

The final approach will be similar to as you've done and only some templates will be affected. Also make sure to keep the order of the js files as they are loaded